src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -953,6 +954,9 @@ def enable_streaming_for_llm_flow(self, stream_required: Callable[[], bool]):
 
         :return: None
         """
+        if not hasattr(self, "_flow"):
```

Comment:
We may implement the method in the sub class instead of using `hasattr`

Reply:
Done